src/server/worker/master.go

```diff
@@ -536,21 +478,14 @@ func (a *APIServer) waitJob(pachClient *client.APIClient, jobInfo *pps.JobInfo,
 
 	backoff.RetryNotify(func() (retErr error) {
 		// block until job inputs are ready
+		// TODO(bryce) This should be removed because it is no longer applicable.
```

Comment:
Remove this?

Reply:
I cannot for the life of me remember why I put this here lol. I vaguely remember discussing with you something that suggested to me that I should remove this a while ago, but looking at it recently I cannot remember why that made sense. Should I just remove the comment?